src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -118,7 +128,7 @@ const renderQuickMarcEditor = (props) => (render(
           deleteRecord: jest.fn(),
           moveRecord: jest.fn(),
         }}
-        initialValues={initialValues}
+        initialValues={props?.initialValues ? props.initialValues : initialValues}
```

Comment:
I don't think this is needed - if there's `props.initialValues` it'll be destructured on line 134 and will override this default value